sources/repo-update

```diff
@@ -154,11 +133,69 @@ fi
 [ -e "${LOCKFILE}" ] && error_mes "lock"
 /usr/bin/touch "${LOCKFILE}" 2> /dev/null || /usr/bin/true
 
-## work block
-# building
+## Work block
+# Building
 echo -e "${bwhite}[II]${cclose} Building packages"
 cd "${STAGINGDIR}"
-/usr/bin/find -name 'PKGBUILD' -type f -execdir /usr/bin/bash -c "func_build "${PREPAREDIR}" "${ROOTDIR}"" \;
+#Get package list in staging folder
+for PACKAGE in $(/usr/bin/find -name 'PKGBUILD' -type f); do
+    cd "${STAGINGDIR}"
+    #Get package folder
+    cd $(dirname "${PACKAGE}")
+    #Get PKGBUILD
+    source PKGBUILD
+    echo -e "${bwhite}[II] ${bblue}=>${cclose} Building ${bwhite}${pkgname}${cclose}"
+    #Build for repo architecture
+    for ARCH in ${ARCHS}; do
+    	#Build for package architecture
+	for _ARCHS in ${arch[@]}; do
```

Comment:
it would be great if this cycle will be moved to the separate function as it is now